### PR TITLE
thank you

### PR DESCRIPTION
**Pull request by royaallam**
thank you

### CODE REVIEW

File: HW-Week2_Day03_Inheritance_and_ClassTypes.iml [hunk 1]
```diff
@@ -1,12 +1,79 @@
...
+  }
+
+//عملنا كلاس Rectangleيورث منShapeنورث  properties حقتو
+     class Rectangle:Shape("Rectangle","Blue"){
+    //عرفنا متغريين واعطينهم  doubleواعطينهم قيمة
+         var widthNum:Double=30.0
+         var heightNumber:Double=20.0
+    //عملنا معادلة  الرياضية حطينهافي متغير
+         var resultRec=widthNum*heightNumber
+    //  عملناfunترجع double بطريقة override نورث ويرجع متغير
```
Comment (shuruqt1000) [inline HW-Week2_Day03_Inheritance_and_ClassTypes.iml:36]: what did we inherit ? why is it important and how it affect the rest of your code ?

File: HW-Week2_Day03_Inheritance_and_ClassTypes.iml [hunk 2]
```diff
@@ -1,12 +1,79 @@
...
-    <orderEntry type="inheritedJdk" />
-    <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="library" name="KotlinJavaRuntime" level="project" />
-  </component>
-</module>
\ No newline at end of file
+//عملنا كلاس وخليناهopen نورث منو ونعمل  2-properties
+  open class Shape(var shapeName:String,var shapeColor:String){
+// عملنا funوسميناهاinfo وترجع two partmaterهيا shaper name,shape color
+       fun info(): String {
```
Comment (shuruqt1000) [inline HW-Week2_Day03_Inheritance_and_ClassTypes.iml:4]: why did you use open? what's the importance of it ? why the other once aren't all open explain your code

File: HW-Week2_Day03_Inheritance_and_ClassTypes.iml [hunk 3]
```diff
@@ -1,12 +1,79 @@
...
+  class Triangle():Shape("Triangle","Red"){
+    //عرفنا متغريين واعطينهم  doubleواعطينهم قيمة
+     var baseNum:Double=10.0
+     var heightNum:Double=20.0
+    //عملنا معادلة  الرياضية حطينهافي متغير
+     var resultTri=(heightNum*baseNum)/2
+
+    //  عملناfunترجع double بطريقة override نورث ويرجع متغير
+     override fun computerArea(): Double {
+         return resultTri
```
Comment (shuruqt1000) [inline HW-Week2_Day03_Inheritance_and_ClassTypes.iml:24]: what are we overriding here ?
what part of the code dose it affect ?

**Review by shuruqt1000 — CHANGES_REQUESTED** [review]
please see my comments

File: HW-Week2_Day03_Inheritance_and_ClassTypes.iml (hunk 2 above)
Comment (royaallam) [reply HW-Week2_Day03_Inheritance_and_ClassTypes.iml:4]: to make inheritance from  shape

File: HW-Week2_Day03_Inheritance_and_ClassTypes.iml (hunk 3 above)
Comment (royaallam) [reply HW-Week2_Day03_Inheritance_and_ClassTypes.iml:24]: call it in shape class

File: HW-Week2_Day03_Inheritance_and_ClassTypes.iml (hunk 1 above)
Comment (royaallam) [reply HW-Week2_Day03_Inheritance_and_ClassTypes.iml:36]: to make inherit  from  shape

**Comment by shuruqt1000** [comment]
completed

**Pull request closed without merging**